test/linalg/lu.jl

```diff
@@ -0,0 +1 @@
+@test logdet(Complex64[1.0f0 0.5f0; 0.5f0 -1.0f0]) === 0.22314355f0 + 3.1415927f0im
```

Comment:
Would be worth it if there were more tests, which there should be. So maybe leave it in anticipation of more LU tests?